python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
`makeGangaListByRef` shouldn't change anything about the reference objects as it is copying *by ref*. Now, I think that because of this `__deepcopy__` should be changed back to call `makeGangaList` instead. It was changed to copy by ref in https://github.com/ganga-devs/ganga/commit/72e674ebb6858074fd9de975537a5a45507ee278#diff-49914f23e50f4cbb9e935817edbb86a4R289 but I don't that this is what we want.